OrbitCore/Pdb.h

```diff
@@ -38,7 +38,7 @@ class Pdb {
   orbit_client_protos::FunctionInfo* GetFunctionFromExactAddress(uint64_t a_Address);
   orbit_client_protos::FunctionInfo* GetFunctionFromProgramCounter(uint64_t a_Address);
 
-  void ProcessData();
+  void ProcessData(class Process* process);
```

Comment:
Why not just `void ProcessData(Process* process);`

Reply:
Then the headers would depend cyclicly from each other.

Reply:
To avoid using "class Process* process" in the .cpp as well (as it already includes OrbitProcess.h), would it be possible to use a forward declaration in the .h so you can declare the method as ProcessData(Process* process) instead?

Reply:
Thanks for the suggestions. As discussed with Irina offline, I have removed the dependency to Process in Pdb completely and added a AddFunctions method to the process.